template/python3-flask/index.py

```diff
@@ -22,7 +23,13 @@ def fix_transfer_encoding():
 @app.route("/", defaults={"path": ""}, methods=["POST", "GET"])
 @app.route("/<path:path>", methods=["POST", "GET"])
 def main_route(path):
-    ret = handler.handle(request.get_data())
+    raw_body = os.getenv("RAW_BODY")
+
+    as_text = True
+    if raw_body == "True":
```

Comment:
perhaps this would be useful here https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

> Convert a string representation of truth to true (1) or false (0).
>
> True values are y, yes, t, true, on and 1; false values are n, no, f, false, off and 0. Raises ValueError if val is anything else.

Reply:
Yes sounds reasonable to me